README.md

```diff
@@ -144,10 +144,12 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
 * What tools and techniques do you use debugging JavaScript code?
 * What language constructions do you use for iterating over object properties and array items?
+* Explain destructuring assignment.
 * Explain the difference between mutable and immutable objects.
   * What is an example of an immutable object in JavaScript?
   * What are the pros and cons of immutability?
   * How can you achieve immutability in your own code?
+* What is a pure function? What are the side effects of a pure function?
```

Comment:
Maybe, but that feels like a "general coding" question. Not saying it doesn't belong, but the JS list is currently quite focused on JS/DOM-specific concepts. The closest it comes to "general" questions AFAICT is the sync/async function question and even that one is most relevant to JS since its symmetric treatment of async and sync code at a syntax level is pretty unusual.